src/main/java/org/broadinstitute/consent/http/service/UserService.java

```diff
@@ -84,6 +84,8 @@ public User findUserById(Integer id) throws NotFoundException {
         if (user == null) {
             throw new NotFoundException("Unable to find user with id: " + id);
         }
+        List<LibraryCard> cards = libraryCardDAO.findLibraryCardsByUserId(user.getDacUserId());
+        user.setLibraryCards(cards);
```

Comment:
This works, but it would be more efficient to pull the LC info from the `userDAO.findUserById(id);` call so we don't have to make the extra db query.

Reply:
Conversation resolved in slack. The nested institution subqueries make this onerous.